src/contracts/proxies/ODSafeManager.sol

```diff
@@ -191,8 +195,7 @@ contract ODSafeManager is IODSafeManager, Authorizable, Modifiable {
       _sData.collateralType, _sData.safeHandler, collateralSource, debtDestination, _deltaCollateral, _deltaDebt
     );
 
-    vault721.updateVaultHashState(_safe);
-
+    vault721.updateNfvState(_safe);
```

Comment:
We may need to add a check here to only updateNfvState if the value is decreased, to avoid DoS